### DISCUSSION TITLE
Find the odd one 🔍

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1250] [Galaxy Trivia] [answered]
**So I'm cuter**
**Sweaty ken**
**He'll mesh crooks**
**Narcoleptic**

**Comment by peterbaccaro** [comment, marked as answer]
1. TOM CRUISE
2. KANYE WEST
3. SHERLOCK HOLMES
4. ERIC CLAPTON

The odd one out is **SHERLOCK HOLMES** as he is a fictional detective and the others are all celebrities.
